core/arch/arm/kernel/stmm_sp.c

```diff
@@ -578,8 +578,8 @@ static void save_sp_ctx(struct stmm_ctx *spc, struct thread_svc_regs *svc_regs)
 }
 #endif
 
-static bool return_helper(bool panic, uint32_t panic_code,
-			  struct thread_svc_regs *svc_regs)
+static void return_nsec_helper(bool panic, uint32_t panic_code,
```

Comment:
maybe misnamed. `return_sp_client_helper()`?  keep `return_helper()`?

Reply:
If this needs to be renamed i guess `return_from_sp_helper()` would be more accurate.